diff --git a/modules/custom/custom_patch/abc.php b/modules/custom/custom_patch/abc.php
index 2f1c0721..3a6f96f6 100644
--- a/modules/custom/custom_patch/abc.php
+++ b/modules/custom/custom_patch/abc.php
@@ -1,5 +1,5 @@
 name: custom_hook
 type: module
-description: This Module Provides  additional functionality for the site.
+description: Provides  additional functionality for the site.
 package: Custom
 core_version_requirement: ^9 || ^10
